GOVERNANCE.md

```diff
@@ -0,0 +1,94 @@
+# Overview 
+
+Apollo is a meritocratic, consensus-based community project. Anyone with an interest in the project can join the community, contribute to the project design and participate in the decision-making process.
+
+This document describes how that participation takes place and how to set about earning merit within the project community.
+
+# Roles and Responsibilities
+
+Apollo community is composed of and operated by the following roles:
+
+- Users
+- Contributors
+- Committers
+- Project Management Committee (PMC)
+
+## Users
+
+Users are community members who have a need for the project. They are the most important members of the community and without them the project would have no purpose. Anyone can be a user and there are no special requirements.
+
+## Contributors
+
+Contributors are community members who contribute in concrete ways to the project.
+
+### How to become a Contributor
+
+- merged at least 1 pull request
+
+You are also encouraged to participate in the projects in the following ways:
+
+- Actively answer technical questions raised by community users in GitHub issues.
+- Help test the projects
+- Help review the pull requests (PRs) submitted by others
+- Help improve technical documents
+- Submit valuable issues
+- Report or fix known and unknown bugs
+- Write articles about source code analysis and usage cases for a project.
+- Give representations of Apollo topic in conferences.
+- Take part in our discussions of features, enhancements, etc.
+
+## Committers
+
+Committers are contributors who have shown that they are committed to the continued development of the project through ongoing engagement with the community and recognized by PMCs for their outstanding contributions.
+
+### How to become a Committer
```

Comment:
If become a committer, should i sign a agreement, like ICLA?

Reply:
Good point! I raised a #3678 for this.

Reply:
CLA mechanism was introduced by #3683